vendor/assets/javascripts/jquery.textile.js

```diff
@@ -27,7 +27,7 @@
         resize: true,
         // HTML templates
         tpl: {
-          wrap: '<div class="textile-wrap"><ul class="textile-toolbar"></ul><div class="textile-inner row"></div></div>',
+          wrap: '<div class="textile-wrap"><ul class="textile-toolbar"></ul><div class="textile-inner row" data-id="textile-inner"></div></div>',
```

Comment:
using an "ID" as a front-end behaviour is against the style guide.

Reply:
Resolved with new entry point changes.